inc/database/class-mysql-processor.php

```diff
@@ -37,9 +37,30 @@ public function get_connection() {
 
 		// Uncomment to enable connection debugging.
 		$connection->setAttribute( \PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION );
+
+		$this->set_mysql_global_defaults( $connection );
 		return $connection;
 	}
 
+	/**
+	 * Set any global MySQL default settings and customizations
+	 * for this session.
+	 *
+	 * @param    PDO   $connection Current PDO connection
+	 *
```

Comment:
cut this extra line? standard doctoc?

Reply:
hmm, i always thought `@return` in it's own grouping (space between params and return) http://docs.phpdoc.org/getting-started/your-first-set-of-documentation.html#example